openssl/Cargo.toml

```diff
@@ -24,6 +24,7 @@ cfg-if = "0.1"
 foreign-types = "0.3.1"
 lazy_static = "1"
 libc = "0.2"
+paste = "0.1.5"
```

Comment:
This increases the minimum supported rust version, so we'll probably want to just avoid it for now.